aiosmtpd/smtp.py

```diff
@@ -383,10 +384,14 @@ def _set_rset_state(self):
         """Reset all state variables except the greeting."""
         self._set_post_data_state()
 
-    async def push(self, status):
-        response = bytes(
-            status + '\r\n', 'utf-8' if self.enable_SMTPUTF8 else 'ascii')
-        self._writer.write(response)
+    async def push(self, status: AnyStr):
+        if isinstance(status, str):
+            response = bytes(
+                status, 'utf-8' if self.enable_SMTPUTF8 else 'ascii')
+        else:
+            response = status
```

Comment:
`SMTP.push()` is now a bit smarter and can handle `str` or `bytes`.

After all, internally it performs a str.encode() so why not enable it to handle bytes directly?